mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -391,6 +397,7 @@ func flattenBigtableCluster(c *bigtable.ClusterInfo) map[string]interface{} {
 		autoscaling_config[0]["min_nodes"] = c.AutoscalingConfig.MinNodes
 		autoscaling_config[0]["max_nodes"] = c.AutoscalingConfig.MaxNodes
 		autoscaling_config[0]["cpu_target"] = c.AutoscalingConfig.CPUTargetPercent
+		autoscaling_config[0]["storage_target"] = c.AutoscalingConfig.StorageUtilizationPerNode
```

Comment:
nit: Should we be more consistent with the naming? "storage_target" vs "StorageUtilizationPerNode"? Or you actually prefer the new name "storage_target"?

Reply:
The name `StorageUtilizationPerNode` is inconsistent but it was in go client and already published. The terraform field `storage_target` is consistent with `cpu_target`.